filebeat/harvester/reader/log.go

```diff
@@ -28,13 +28,13 @@ type logFileReader struct {
 }
 
 type LogFileReaderConfig struct {
-	CloseOlder         time.Duration
-	BackoffDuration    time.Duration
-	MaxBackoffDuration time.Duration
-	BackoffFactor      int
-	CloseRenamed       bool
-	CloseRemoved       bool
-	CloseEOF           bool
+	Backoff       time.Duration
+	MaxBackoff    time.Duration
+	BackoffFactor int
```

Comment:
This is what we currently have in libbeat: https://github.com/elastic/beats/blob/master/libbeat/common/backoff.go